egs/wsj/s5/steps/nnet3/dump_bottleneck_features.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+# This script dumps bottleneck feature for model trained using nnet3.
+
+# Begin configuration section.
+feat_type=
+stage=1
+nj=4
+cmd=run.pl
+use_gpu=false
+bnf_name=renorm4
+use_ivector=false
+ivector_dir=
+# Begin configuration.
+transform_dir=
+# End configuration options.
+
+echo "$0 $@"  # Print the command line for logging
+
+[ -f path.sh ] && . ./path.sh # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# != 5 ]; then
```

Comment:
As I had suggested in other scripts, it might be better to support multiple argument lists like `steps/mke_mfcc.sh` for the sake of consistency. By default you could drop the arkdir and logdir arguments.